llvm/lib/MCCAS/MCCASDebugV1.cpp

```diff
@@ -255,6 +255,14 @@ void AbbrevEntryWriter::writeAbbrevEntry(DWARFDie DIE) {
     if (Form == dwarf::Form::DW_FORM_strp)
       Form = dwarf::Form::DW_FORM_strp_cas;
     writeULEB128(Form);
+    // Dwarf 5: Section 7.4:
+    // The attribute form DW_FORM_implicit_const is another special case. For
+    // attributes with this form, the attribute specification contains a third
+    // part, which is a signed LEB128 number. The value of this number is used
+    // as the value of the attribute, and no value is stored in the .debug_info
+    // section.
```

Comment:
I think you may need to paraphrase the text here since the text you're quoting is under a license that may not be compatible with the LLVM one.